src/FakeItEasy/Creation/CastleDynamicProxy/CastleDynamicProxyInterceptionValidator.cs

```diff
@@ -34,7 +34,7 @@ private static string GetReasonForWhyMethodCanNotBeIntercepted(MethodInfo method
                 var explicitImplementation = method.Name.Contains('.');
                 if (!explicitImplementation)
                 {
-                    return "Sealed methods can not be intercepted.";
+                    return "Method should be marked as virtual, abstract or part of an Interface in oreder to be intercepted.";

```

Comment:
Cool!
Since you're invested, and have already started modifying code and tests, are you interested in updating your PR accordingly? After all, it was your feedback that will lead to an improvement one way or another!